targets/unity-v2/make.js

```diff
@@ -378,7 +418,11 @@ function getAuthParams(apiCall) {
     return "AuthType.None";
 }
 
-function getRequestActions(tabbing, apiCall) {
+function getInstanceParams(apiCall) {
+    return "authenticationContext, apiSettings"
+}
```

Comment:
Please remove this method entirely as it seems we don't need it at all. Just use those two parameters directly in template